docs/core/config/auth/passdb.md

```diff
@@ -442,6 +445,35 @@ a load spike of everybody getting logged in at exactly the same time.
 
 Do not perform any authentication, just store extra fields if user is found.
 
+##### `check_client_fp`
+
+Match client certificate or public key fingerprint. See [[setting,ssl_peer_certificate_fingerprint_hash]]
+
+This is intended to be used instead of verifying certificates with CA certificates. If you are using CA certificates,
+you can only fail authentication if the value is non-empty and does not match. If CA certificates are not used, then
+if no passdb matches the fingerprint, then authentication is failed.
+
+See [[setting,ssl_server_request_client_cert]] on how to allow any certificate.
```

Comment:
Maybe we need a clear bulletin point list what is required for this to work as expected:

 * [[setting,auth_ssl_require_client_cert,yes]]
 * [[setting,ssl_server_request_client_cert,any-cert]] to skip CA certificate checks
 * passdb `check_client_fp`, `check_client_cert_fp` or `check_client_publkey_fp` where the value contains the user's client certificate's fingerprint